homeassistant/components/upcloud/__init__.py

```diff
@@ -82,6 +82,7 @@ def upcloud_update(event_time):
         dispatcher_send(hass, SIGNAL_UPDATE_UPCLOUD)
 
     # Call the UpCloud API to refresh data
+    upcloud_update(datetime.now())
```

Comment:
Either use `None` (since you're not using the data) or use `utcnow()` imported from `homeassistant.util.dt` to correctly mimic `track_time_interval`.